router/pump.go

```diff
@@ -361,10 +362,15 @@ func newContainerPump(container *docker.Container, stdout, stderr io.Reader) *co
 				}
 				return
 			}
+			logMessage, logTime, err := parseLogLine(line)
```

Comment:
`parseLogLine` should only be called if this feature is enabled. Also, if we get an error, perhaps we should fall back to default behaviour of using `Time.Now()`